.github/workflows/ci.yml

```diff
@@ -1,17 +1,23 @@
-name: Besom
+name: Besom build and test
 
 on:
   push:
     branches: [ 'main', 'develop' ]
     paths-ignore: 
     - '**.md'
     - 'LICENSE'
+    - 'template'
```

Comment:
Actually it would be good to have tests that validate that our templates at least compile, but that could be added later

Reply:
Agreed, https://github.com/VirtusLab/besom/issues/155